### PR TITLE
Hw 03 sixth part

### PR DESCRIPTION
**Pull request by Alex-Korzun**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/cars/all.jsp [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                <c:out value="${car.drivers}"/>
+            </td>
+            <td>
+                <a href="${pageContext.request.contextPath}/cars/delete?id=${car.id}">Delete</a>
+            </td>
+        </tr>
+    </c:forEach>
+</table>
+</body>
+</html>
```
Comment (vlykhvar) [inline src/main/webapp/WEB-INF/views/cars/all.jsp:38]: need to add empty line in the end of all classes and files

**Review by vlykhvar — CHANGES_REQUESTED** [review]

File: src/main/webapp/WEB-INF/views/cars/all.jsp (hunk 1 above)
Comment (Alex-Korzun) [reply src/main/webapp/WEB-INF/views/cars/all.jsp:38]: emptyline

Comment (Alex-Korzun) [reply src/main/webapp/WEB-INF/views/cars/all.jsp:38]: I have them in every file) I have checked this option very carefully

**Review by vlykhvar — APPROVED** [review]

**Review by KseniiaMakarova — APPROVED** [review]
great job!

**Pull request merged**